frontend/routes/Wifi.jsx

```diff
@@ -12,42 +29,235 @@ export class Component extends React.Component {
+                active={orderBy === key}
+                direction={orderBy === key ? order : 'asc'}
+                onClick={createSortHandler(key)}
+              >
+                <b>{headCells[key].label}</b>
+                {orderBy === key ? (
+                  <span className={classes.visuallyHidden}>
+                    {order === 'desc' ? 'sorted descending' : 'sorted ascending'}
+                  </span>
+                ) : null}
+              </TableSortLabel>
+            </TableCell>
+          );
+        })}
+      </TableRow>
+    </TableHead>
+  );
+}
+
+EnhancedTableHead.propTypes = {
+  classes: PropTypes.object.isRequired,
+  onRequestSort: PropTypes.func.isRequired,
+  order: PropTypes.oneOf(['asc', 'desc']).isRequired,
+  orderBy: PropTypes.string.isRequired,
+};
+
+const useToolbarStyles = makeStyles(theme => ({
+  root: {
+    padding: theme.spacing(2),
+  },
+  highlight:
+    theme.palette.type === 'light'
+      ? {
+          color: theme.palette.secondary.main,
+          backgroundColor: lighten(theme.palette.secondary.light, 0.85),
+        }
+      : {
+          color: theme.palette.text.primary,
+          backgroundColor: theme.palette.secondary.dark,
+        },
+  title: {
+    flex: '1 1 100%',
+  },
+}));
+
+const EnhancedTableToolbar = () => {
+  const classes = useToolbarStyles();
+  return (
+    <Box>
+    <Toolbar>
+      <Typography className={classes.title} variant="h6" id="tableTitle" component="span">
+      Active Wifi Networks
+      </Typography>      
+    </Toolbar>
+
+    </Box>
+
+  );
+};
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '90%',
+    margin: theme.spacing(2),
+  },
+  paper: {
+    width: '100%',
+    padding: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+  },
+  table: {
+    minWidth: 550,
+  },
+  visuallyHidden: {
+    border: 0,
+    clip: 'rect(0 0 0 0)',
+    height: 1,
+    margin: -1,
+    overflow: 'hidden',
+    padding: 0,
+    position: 'absolute',
+    top: 20,
+    width: 1,
+  },
+}));
+
+function descendingComparator(a, b, orderBy) {
+  if (headCells[orderBy].numeric) {
+    return parseInt(b[orderBy]) < parseInt(a[orderBy]) ? -1 : parseInt(b[orderBy]) > parseInt(a[orderBy]);
+  } else {
+    return a[orderBy] < b[orderBy] ? -1 : a[orderBy] > b[orderBy];
+  }
+}
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+function stableSort(array, comparator) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) return order;
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map(el => el[0]);
+}
+
+function EnhancedTable(props) {
+  const classes = useStyles();
+  const [order, setOrder] = useState('asc');
+  const [orderBy, setOrderBy] = useState(first_column);
```

Comment:
Why use a constant for `first_column` and not for `'asc'`. Please be consistent in the code, I think here using a constant just for `first_column` is unnecessary.

Reply:
removed first_column, Created constant for initial direction('asc') as well.